reference/OpenAPI.json

```diff
@@ -11533,7 +11528,8 @@
       "5_req_create_publication_campaign": {
```

Comment:
Może lepiej od razu wywalić bo z tego co widzę to ten oraz _name i _id nie są używane wgl.

Reply:
Nie chce teraz tego robić, czasem stare modele się przydają w pisaniu nowych descriptions.
Zrobie PR-ke jutro usuwając stare - nieużywane modele.